<sdf version="1.5">
  <world name="default" default_physics="preset_1">
    <physics name="preset_1" type="ode">
      <!--<max_step_size>0.01</max_step_size>
      <max_contacts>21</max_contacts>
      <ode>
        <solver>
          <min_step_size>0.001</min_step_size>
          <iters>50</iters>
          <sor>1.4</sor>
          <use_dynamic_moi_rescaling>1</use_dynamic_moi_rescaling>
        </solver>
        <constraints>
          <cfm>0.01</cfm>
          <erp>0.3</erp>
          <contact_max_correcting_vel>200</contact_max_correcting_vel>
          <contact_surface_layer>0.002</contact_surface_layer>
        </constraints>
      </ode>
      <simbody>
        <accuracy>0.01</accuracy>
        <max_transient_velocity>0.01</max_transient_velocity>
      </simbody>
      <bullet>
        <solver>
          <min_step_size>0.001</min_step_size>
          <iters>50</iters>
          <sor>1.4</sor>
        </solver>
        <constraints>
          <cfm>0.01</cfm>
          <erp>0.3</erp>
          <contact_surface_layer>0.002</contact_surface_layer>
          <split_impulse>0</split_impulse>
          <split_impulse_penetration_threshold>-0.02</split_impulse_penetration_threshold>
        </constraints>
      </bullet>-->
    </physics>

    <!--<physics name="preset_2" type="simbody">
      <max_step_size>0.02</max_step_size>
      <max_contacts>42</max_contacts>
      <ode>
        <solver>
          <min_step_size>0.002</min_step_size>
          <iters>100</iters>
          <sor>1.5</sor>
          <use_dynamic_moi_rescaling>0</use_dynamic_moi_rescaling>
        </solver>
        <constraints>
          <cfm>0.02</cfm>
          <erp>0.6</erp>
          <contact_max_correcting_vel>400</contact_max_correcting_vel>
          <contact_surface_layer>0.004</contact_surface_layer>
        </constraints>
      </ode>
      <simbody>
        <min_step_size>0.002</min_step_size>
        <accuracy>0.02</accuracy>
        <max_transient_velocity>0.02</max_transient_velocity>
      </simbody>
      <bullet>
        <solver>
          <min_step_size>0.002</min_step_size>
          <iters>100</iters>
          <sor>1.5</sor>
        </solver>
        <constraints>
          <cfm>0.02</cfm>
          <erp>0.6</erp>
          <contact_surface_layer>0.004</contact_surface_layer>
          <split_impulse>1</split_impulse>
          <split_impulse_penetration_threshold>-0.04</split_impulse_penetration_threshold>
        </constraints>
      </bullet>
    </physics>-->

  </world>
</sdf>
